felds/quick-mailer-bundle/4.0/config/packages/quickmailer.yaml

```diff
@@ -0,0 +1,13 @@
+# For the full configuration, visit:
```

Comment:
Could you rename this file to felds_quickmailer.yaml?

Reply:
then will we have to change the config root to `felds_quickmailer` too?

Reply:
No no. Just the file name

Reply:
I find that a little misleading. Are there many packages using a file name different that the config root?
I assume the same conditions that could lead to a filename conflict would also break the config…

Reply:
The config root has really nothing to do with this. The file name of the config file (in the recipe) should be prefixed with the organization. 

I see that this is not documented anywhere, but this is a practice done by many (most?) packages.

Reply:
good to know :)

I'll fix this in a minute